app/src/main/java/com/fstyle/androidtrainning/screen/main/MainContract.java

```diff
@@ -1,5 +1,6 @@
 package com.fstyle.androidtrainning.screen.main;
 
+import com.fstyle.androidtrainning.data.local.roomdb.entity.TrackEntity;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.data.local.roomdb.entity.TrackEntity. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports